style.css

```diff
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+  font-family: "Courier New", Courier, monospace;
+  background-image: url('https://static.wixstatic.com/media/c88f3c_80f37c005f9a46209de3525512af4e1d~mv2.gif');
+  background-size: cover;
+}
+
+p {
+  margin: 1px;
+  font-weight: 300;
+  background-color: #fff;
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+}
+
+.cards {
+  margin: 50px auto;
+  width: 90%;
+  height: 600px;
+  background-image: url('https://i.pinimg.com/736x/71/84/8f/71848fd22b78ca45b3dd628888f6b866.jpg');
+  background-size: cover;
+  opacity: 0.9;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border-radius: 15px;
+}
+
+.killer-deck {
+
```

Comment:
Maybe good to remove this whitespace area so we can see easier that this is all together?